mPowerSDK/Dashboard/APHDashboardViewController.m

```diff
@@ -295,9 +316,11 @@ - (void)prepareScoringObjects
     self.customScoring.customMaximumPoint = 5.0;
     self.customScoring.caption = NSLocalizedStringWithDefaultValue(@"APH_DAILY_CUSTOM_CAPTION", nil, APHLocaleBundle(), @"Custom Question", @"Dashboard caption for daily user-defined custom question report");
 
-    if (!self.correlatedScoring) {
-        [self prepareCorrelatedScoring];
+    if (!self.correlatedScores) {
+        self.correlatedScores = [NSMutableArray arrayWithArray:@[self.scoreArray[0], self.scoreArray[1]]];
```

Comment:
Very brittle. Someone asked to modify the scoreArray and then you are correlating the wrong data. Just use tapRightScoring and tapLeftScoring directly.